js/app.js

```diff
@@ -0,0 +1,30 @@
+var toDoApp = angular.module('toDoApp', []);
+function ToDoController($scope) {
+  $scope.tasks = [
+    { todo: 'Cleaning', completed: false }, 
+    { todo: 'Shopping', completed: false }, 
+    { todo: 'Laundry', completed: false },
+    { todo: 'Ironing', completed: false }
+  ];
```

Comment:
In JS, `{}` represents an object. The keys can be strings, but if you want it to be JSON, the keys MUST be strings